tests/Tokenizer/Analyzer/Analysis/ArgumentAnalysisTest.php

```diff
@@ -67,7 +67,7 @@ public function testType(): void
     public function testNoTypeFound(): void
     {
         $analysis = new ArgumentAnalysis('$name', 1, null, null);
-        self::assertFalse($analysis->hasDefault());
-        self::assertNull($analysis->getDefault());
+        self::assertFalse($analysis->hasTypeAnalysis());
+        self::assertNull($analysis->getTypeAnalysis());
```

Comment:
why those changes?

Reply:
What's the method name? 

New check `dev-tools/check_duplicate_test_methods.php` reported it - correctly - as a [duplicate](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.59.3/tests/Tokenizer/Analyzer/Analysis/ArgumentAnalysisTest.php#L51-L72) of method `testNoDefaultFound`.

Reply:
why such pushback?
out of the blue i don't even recall what this class does.
and seeing the change in test logic, being pushed as part of "duplicated test methods", was a surprise to me. thus the question.

you do have that knowledge, as just touching it.
please, when reviewer is having no clue what's the reason behind the change you provided and asking for it, and you obviously know the reason, just share it - instead of pushing back "no reviewer, I won't tell you, you need to figure out yourself. I can give you some hints".

Reply:
Not a pushback, I don't know what it does either, I cared only about what is duplicate and what method names are.